From d18845d91d9bdf84ec330459980219522f438a76 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai.pt@gmail.com>
Date: Fri, 2 Aug 2024 17:29:42 +0800
Subject: [PATCH 3/6] ARM: dts: aspeed: catalina: add ipmb-dev node

ipmb-dev node was removed due to compatible string not exist on upstream.
Add back ipmb-dev node back in this patch.

Signed-off-by: Potin Lai <potin.lai.pt@gmail.com>
---
 .../boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts   | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index 7d2e5f5f7cc18..4d7ff91a02a4f 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -680,6 +680,16 @@ rtc@6f {
 
 &i2c7 {
 	status = "okay";
+	bus-frequency = <100000>;
+	multi-master;
+	aspeed,hw-timeout-ms = <1000>;
+
+	//USB Debug Connector
+	ipmb@10 {
+		compatible = "ipmb-dev";
+		reg = <(0x10 | I2C_OWN_SLAVE_ADDRESS)>;
+		i2c-protocol;
+	};
 };
 
 &i2c8 {
-- 
2.31.1

